index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse </title>
+</head>
```

Comment:
- [ ] [REQUIRED] Good job so far! You forgot to link the `style.css` file to the `index.html` file, so the `style` given to the `h1` tag is not working. Please correct it 🙏 

![image](https://user-images.githubusercontent.com/63915024/204869531-0c18d76c-86ea-4612-8de2-d03d588a2488.png)